docs/config/io_helidon_grpc_client_GrpcChannelDescriptor.adoc

```diff
@@ -0,0 +1,54 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+ifndef::rootdir[:rootdir: {docdir}/..]
+:description: Configuration of io.helidon.grpc.client.GrpcChannelDescriptor
+:keywords: helidon, config, io.helidon.grpc.client.GrpcChannelDescriptor
+:basic-table-intro: The table below lists the configuration keys that configure io.helidon.grpc.client.GrpcChannelDescriptor
+include::{rootdir}/includes/attributes.adoc[]
+
+= GrpcChannelDescriptor (grpc.client) Configuration
+
+// tag::config[]
+
+
+Type: link:{javadoc-base-url}/io.helidon.grpc.client/io/helidon/grpc/client/GrpcChannelDescriptor.html[io.helidon.grpc.client.GrpcChannelDescriptor]
+
+
+
+
+== Configuration options
```

Comment:
This is auto-generated by a tool that Tomas created, that's why you would see all files in doc/config to be using the same. However, this files are not standalone docs, but rather are added as includes and hence. would not stay at  level 2. For example, it will be used like this:

include::{rootdir}/config/io_helidon_grpc_core_GrpcTlsDescriptor.adoc[leveloffset=3]

so essentially it gets pushed to become level 5.